lib/components/narrative/line-itin/transit-leg-body.js

```diff
@@ -203,18 +203,14 @@ class AlertsBody extends Component {
           .sort((a, b) => b.effectiveStartDate - a.effectiveStartDate)
           .map((alert, i) => {
             const effectiveStartDate = moment(alert.effectiveStartDate)
-            let effectiveDateString = 'Effective as of '
             const daysAway = moment().diff(effectiveStartDate, 'days')
-            // Add time if alert is effective within one day.
-            if (Math.abs(daysAway) <= 1) {
-              effectiveDateString += `${effectiveStartDate.format(timeFormat)}, `
-            }
-            effectiveDateString += effectiveStartDate
-              .calendar(
-                null,
-                { sameElse: longDateFormat }
-              )
-              .split(' at')[0]
+            // Add time if alert is effective within one day. Otherwise, use
+            // calendar long date format (e.g., July 31, 2019).
+            const dateTimeFormatter = Math.abs(daysAway) <= 1
```

Comment:
This variable isn't a formatter, rather it's just a string. I think it'd make sense to rename this variable to indicate that. Everything else looks good.